code/game/objects/items.dm

```diff
@@ -266,7 +266,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		return 1
 
 /obj/item/blob_act(obj/structure/blob/B)
-	if(B && B.loc == loc)
+	if(B && B.loc == loc && !(resistance_flags & INDESTRUCTIBLE))
 		qdel(src)
```

Comment:
While you're at it, would you consider swapping in a safe access?
```
if(B?.loc == loc && !(resistance_flags & INDESTRUCTIBLE))
```

Reply:
Can B even be null?

Reply:
![image](https://user-images.githubusercontent.com/26465327/172252257-0b8b97a6-4347-4ea1-9370-16f2200db9e5.png)
I'll just fix these 1 off cases actually